app/policies/form_policy.rb

```diff
@@ -58,7 +58,12 @@ def can_add_page_routing_conditions?
     form_has_two_or_more_pages && form_has_qualifying_pages
   end
 
-  alias_method :can_make_form_live?, :can_change_form_submission_email?
+  def can_make_form_live?
+    return can_change_form_submission_email? unless FeatureService.enabled?(:groups)
```

Comment:
We talked about this when developing it and decided to leave it out (on the logic that if this happens something has been migrated wrongly and we want it to get flagged to Sentry). But if it'll get in the way of our testing maybe we should add the check - what do you think @jamie-o-wilkinson?

Reply:
Since everywhere else we're using the feature flag we do this check, we've added it to our new uses too. We've also added TODOs to make sure we remove the check along with the feature flag 👍